### PR TITLE
fix: grep command

### PR DESCRIPTION
**Pull request by alexalex89**
## Checklist
I have:
- [ ] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Sign up on [customer.appsmith.com](https://customer.appsmith.com/) and generate
 5. To retrieve the salt and password from the pod, run the following command and copy the values to `values.yaml`.
 
    ```bash
-   kubectl exec <pod_name> -- grep /appsmith-stacks/configuration/docker.env APPSMITH_ENCRYPTION_
+   kubectl exec <pod_name> -- grep APPSMITH_ENCRYPTION_ /appsmith-stacks/configuration/docker.env
```
Comment (jnikhila) [inline website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:39]: @pratapaprasanna - Please review.

Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:39]: @pratapaprasanna - as we are in the process of rehauling the doc, we will point to appropriate sections in the [Backup and Restore](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl) doc. Please confirm whether the below command should work, or do we need to update it as per this PR?

```
kubectl exec ANY_APPSMITH_POD_NAME cat /appsmith-stacks/configuration/docker.env | grep ENCRYPTION
```

Comment (alexalex89) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:39]: @jnikhila This command should work as well, but you should add a double dash between pod name and command:
```
kubectl exec ANY_APPSMITH_POD_NAME -- cat /appsmith-stacks/configuration/docker.env | grep ENCRYPTION
```
Usage without double dash is deprecated.

Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:39]: Thanks @alexalex89.

**Review by pratapaprasanna — APPROVED** [review]

File: website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md (hunk 1 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md:39]: Thanks, @pratapaprasanna - I'll make these command changes to backup and restore doc in the same PR, as the upgrade to the business edition K8s page will point to these instructions. 
Thanks, @alexalex89 for raising this.

**Comment by jnikhila** [comment]
Closing this PR as the fix will be added as part of #1340

**Pull request closed without merging**